xarray_sentinel/sentinel1.py

```diff
@@ -3,14 +3,28 @@
 import warnings
 
 import numpy as np
+import rioxarray  # type: ignore
 import xarray as xr
 
 from xarray_sentinel import conventions, esa_safe
 from xarray_sentinel.esa_safe import PathType
 
 
-def open_gcp_dataset(product_path: PathType) -> xr.Dataset:
-    geolocation_grid_points = esa_safe.parse_geolocation_grid_points(product_path)
+def read_swath_attributes(annotation_path: PathType) -> T.Dict[str, T.Any]:
```

Comment:
In general the information returned is presente elsewhere, in the numbers and size of the bursts. I would really drop it.

Reply:
Removed in 6ad0e6b47a8648dc9df7adce1abf58294700b821